<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>20</horstretch>
     <verstretch>20</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPlainTextEdit" name="textDisplay">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>820</y>
      <width>1150</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="map">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>100</y>
      <width>1150</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Images/Direction/ZorkPics/map.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="counter">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>10</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="swordBlueprint">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>110</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTreeWidget" name="trapTree_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>665</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <attribute name="headerDefaultSectionSize">
     <number>125</number>
    </attribute>
    <attribute name="headerShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Trap</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Build time left</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Damage</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Location</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="text">
      <string>Spikes</string>
     </property>
     <property name="text">
      <string> 5</string>
     </property>
     <property name="text">
      <string>100</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
   <widget class="QTabWidget" name="inventoryTree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="traps">
     <attribute name="title">
      <string>Traps</string>
     </attribute>
     <widget class="QTreeWidget" name="trapTree">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Trap</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Damage Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Max Damage</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Build time</string>
       </property>
      </column>
     </widget>
     <widget class="QTreeWidget" name="itemDescTree">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>661</width>
        <height>81</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Item Desc</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Trap Total</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="blueprints">
     <attribute name="title">
      <string>Blueprints</string>
     </attribute>
     <widget class="QTreeWidget" name="blueprintTree">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>655</width>
        <height>280</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Trap that can Upgrade</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>More shti?? this is my idea too, cmon</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string>Spikes</string>
       </property>
       <property name="text">
        <string> 5</string>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="place">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>690</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Place Trap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>790</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Destroy Trap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>880</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Upgrade Trap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="talk">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>690</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Talk</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>880</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLabel" name="room01">
    <property name="geometry">
     <rect>
      <x>779</x>
      <y>233</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room02">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>129</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room03">
    <property name="geometry">
     <rect>
      <x>1214</x>
      <y>103</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room04">
    <property name="geometry">
     <rect>
      <x>1469</x>
      <y>129</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room12">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>371</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room13">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>613</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room14">
    <property name="geometry">
     <rect>
      <x>1214</x>
      <y>639</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room05">
    <property name="geometry">
     <rect>
      <x>929</x>
      <y>371</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room06">
    <property name="geometry">
     <rect>
      <x>1094</x>
      <y>276</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room07">
    <property name="geometry">
     <rect>
      <x>1334</x>
      <y>276</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room08">
    <property name="geometry">
     <rect>
      <x>1649</x>
      <y>233</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room09">
    <property name="geometry">
     <rect>
      <x>779</x>
      <y>510</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room10">
    <property name="geometry">
     <rect>
      <x>1094</x>
      <y>484</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room11">
    <property name="geometry">
     <rect>
      <x>1334</x>
      <y>484</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room16">
    <property name="geometry">
     <rect>
      <x>1649</x>
      <y>510</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="room15">
    <property name="geometry">
     <rect>
      <x>1469</x>
      <y>613</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="playerRoom">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>650</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="Down">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/Images/Direction/ZorkPics/downArrow.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Right">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/Images/Direction/ZorkPics/rightArrow.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="Up">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/Images/Direction/ZorkPics/upArrow.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Left">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/Images/Direction/ZorkPics/leftArrow.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="shop">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>380</y>
      <width>182</width>
      <height>157</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trap1">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>740</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trap2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>750</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trap3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>746</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trap4">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>740</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trap5">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>760</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trap6">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>720</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trap7">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>720</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trap8">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>750</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trap10">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>730</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trap9">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>710</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blueprint1">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>740</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blueprint2">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>730</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blueprint3">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>710</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blueprint4">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>740</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blueprint5">
    <property name="geometry">
     <rect>
      <x>1780</x>
      <y>690</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blueprint6">
    <property name="geometry">
     <rect>
      <x>1780</x>
      <y>720</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>map</zorder>
   <zorder>textDisplay</zorder>
   <zorder>counter</zorder>
   <zorder>progressBar</zorder>
   <zorder>trapTree_2</zorder>
   <zorder>inventoryTree</zorder>
   <zorder>place</zorder>
   <zorder>pushButton_13</zorder>
   <zorder>pushButton_14</zorder>
   <zorder>talk</zorder>
   <zorder>pushButton_16</zorder>
   <zorder>swordBlueprint</zorder>
   <zorder>room01</zorder>
   <zorder>room02</zorder>
   <zorder>room03</zorder>
   <zorder>room04</zorder>
   <zorder>room12</zorder>
   <zorder>room13</zorder>
   <zorder>room14</zorder>
   <zorder>room05</zorder>
   <zorder>room06</zorder>
   <zorder>room07</zorder>
   <zorder>room08</zorder>
   <zorder>room09</zorder>
   <zorder>room10</zorder>
   <zorder>room11</zorder>
   <zorder>room16</zorder>
   <zorder>room15</zorder>
   <zorder>playerRoom</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>shop</zorder>
   <zorder>trap1</zorder>
   <zorder>trap2</zorder>
   <zorder>trap3</zorder>
   <zorder>trap4</zorder>
   <zorder>trap5</zorder>
   <zorder>trap6</zorder>
   <zorder>trap7</zorder>
   <zorder>trap8</zorder>
   <zorder>trap10</zorder>
   <zorder>trap9</zorder>
   <zorder>blueprint1</zorder>
   <zorder>blueprint2</zorder>
   <zorder>blueprint3</zorder>
   <zorder>blueprint4</zorder>
   <zorder>blueprint5</zorder>
   <zorder>blueprint6</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
